railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -88,7 +88,13 @@ def self.plugin_rails_init(options)
     # If run via `ruby` we've been passed the files to run directly, or if run
     # via `rake` then they have already been eagerly required.
     unless run_via.ruby? || run_via.rake?
-      ::Rails::TestRequirer.require_files(options[:patterns])
+      # If there are no given patterns, we can assume that the user
+      # simply runs the `bin/rails test` command without extra arguments.
+      if options[:patterns].empty?
+        ::Rails::TestRequirer.require_files(options[:patterns], ["test/system/*_test.rb"])
```

Comment:
I think this (and its friend) wants a `/**/` segment too, to consider tests in subdirectories. We can probably just go with `test/system/**/*`, actually...  no need for the more specific pattern that could get out of sync.